indexer/categories_holder.cpp

```diff
@@ -143,6 +143,7 @@ void ProcessCategory(string const & line, vector<string> & groups, vector<uint32
 // static
 int8_t const CategoriesHolder::kEnglishCode = 1;
 int8_t const CategoriesHolder::kUnsupportedLocaleCode = -1;
+uint8_t constexpr CategoriesHolder::kMaxSupportedLocaleIndex;
```

Comment:
Please, be consistent - use const or constexpr, but don't mix them.

Reply:
done